tox.ini

```
@@ -37,4 +42,4 @@ lines_between_sections = 0
 profile = black
 reverse_relative = True
 sort_relative_in_force_sorted_sections = True
-src_paths = src
+src_paths = src tests
```

This is incorrect; revert.

---

why incorrect -- shouldn't imports be consistently sorted in the tests too?

---

This has nothing to do with whether imports are sorted in the tests.  `src_paths = tests` declares that any modules immediately within `tests/` are to be treated as first-party packages for sorting purposes, but we shouldn't be importing anything from modules within `tests/` anyway.

---

Great, thanks for the explanation!

```suggestion
src_paths = src
```